From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas R <nicolas.org>
Date: Tue, 25 Aug 2020 12:04:23 -0600
Subject: [PATCH] t/lib/Role/Time.pm

---
 t/lib/Role/Time.pm | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/t/lib/Role/Time.pm b/t/lib/Role/Time.pm
index 5a93ac0..5ca2863 100644
--- a/t/lib/Role/Time.pm
+++ b/t/lib/Role/Time.pm
@@ -1,5 +1,6 @@
 package Role::Time;
 
+use v5;
 use strict;
 use warnings;
 
@@ -7,4 +8,4 @@ use Simple::Accessor qw{date};
 
 sub _build_date { '20210102' }
 
-1;
\ No newline at end of file
+1;
